commands/configure.ts

```diff
@@ -111,6 +113,40 @@ export default class Configure extends BaseCommand {
     this.logger.action('update .adonisrc.json file').succeeded()
   }
 
+  /**
+   * Install packages using the correct package manager
+   * You can specify version of each package by setting it in the
+   * name like :
+   *
+   * ```
+   * installPackages(['@adonisjs/lucid@next', '@adonisjs/auth@3.0.0'])
+   * ```
+   */
+  async installPackages(packages: { name: string; isDevDependency: boolean }[]) {
+    const appPath = fileURLToPath(this.app.appRoot)
```

Comment:
You can use `app.makeURL()` to get URL of the application root

Reply:
What i need here is the path to the application root, not a URL
Will replace it with app.makePath